ONSdigital/eq-questionnaire-runner
# Add newrelic to k8s deployment

# Context
Adds NewRelic variables to k8s deployment and the Concourse deploy ci task

# How to review 
- Export the NewRelic environment variables
- Run the `deploy_app.sh` script and check the variables are set in the GCP Kubernetes UI 
- Run a one off deploy via Concourse using the updated ci task and check the variables are set in the GCP Kubernetes UI
